### ISSUE TITLE
Corsair AF3 BCNM fight requiring all members to have key item

### ISSUE THREAD

**Issue by Yaacha123** [EdenServer/community#2780]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x] I have provided my client version in the details. (type /ver into your game window)

### Details

Version: 30181205_0

Attempted the Corsair AF3 (Against All Odds) BCNM fight with 3 members, party leader being the only one with the key item (Life Float) and currently on the quest. 

After the party leader with the key item triggered the cut-scene and the boat started to sail, all remaining party members without the key item entered the BCNM and were hit with a black screen, requiring them to crash the game to resume playing (not in the BCNM, just play at all). These members were playing @30 frames, and one of them confirmed to have a matching version number with the party leader of 30181205_0 who got in successfully with the key item.

A ticket was submitted and a GM contacted us, Pyke got us to re-attempt the BCNM and provided himself with a key item to test the theory and confirmed it as only the party leader and him made it onto the boat and did not receive a black screen, while the other two members experienced the same issue.

**Comment by greyenvy** [comment]
While doing this fight yesterday I encountered this.  I had already killed the WAR mob so just the BLU was left.  I was kiting him and when I went to re-apply Gravity while it was bound, this spammed my log for about two scroll's worth.  
![Ashton_2020 12 02_153324](https://user-images.githubusercontent.com/48332558/101069908-a40b3180-3568-11eb-8dc0-ba9d0ef986f5.png)
Same client version as above.  I did not have the issue with the key item however as Ashton did not have it, but Jorildyn did and was able to get me into the battlefield.

However, when I cleared the battlefield, the ending cutscene played twice for me.  I was in 30 FPS mode already so that certainly wasn't part of it.
